code/online_challenges/src/project_euler/problem_025/problem_025.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+int main() 
+{
+    vector < int > temp, prevFibonacci, currFibonacci;
+
+    int count = 2;
+
+    prevFibonacci.push_back(1);
+    currFibonacci.push_back(1);
+        
```

Comment:
You might want to consider a call to [`std::vector::reserve`](http://en.cppreference.com/w/cpp/container/vector/reserve) for 1000 elements before the loop, so that the `push_back` calls are O(1).